dotnet/src/libraries/AuthorizedStore.Fake/Stores/StoreService.cs

```diff
@@ -23,6 +25,9 @@ public async Task<Store> CreateAsync(Store entity)
             StringExtensions.CheckNullOrWhiteSpace(entity?.ContractContent, nameof(Store.ContractContent));
             CheckInvalidName(entity?.Name);
             await CheckDuplicate(entity?.Name);
+            entity.Category = entity.Category?.Id != null
```

Comment:
```suggestion
            entity.Category = entity.Category != null
```
判斷至 `entity.Category` 即可， `int` 不會為 null。